src/earn/PoolCard.tsx

```diff
@@ -226,26 +188,17 @@ const styles = StyleSheet.create({
   },
   valueTextBold: {
     color: Colors.black,
-    ...typeScale.labelSemiBoldLarge,
+    ...typeScale.labelSemiBoldSmall,
     lineHeight: 20,
```

Comment:
is this still needed?

Reply:
looks the same without it, removed in [0ced40b](https://github.com/valora-inc/wallet/pull/5787/commits/0ced40bb7aebc8de50b2af82141145ff9bf64ef2)